Jenkinsfile.oke

```diff
@@ -281,7 +281,6 @@ EOF
                         DOCKERHUB_PASSWORD = credentials("${dockerhub_password_creds}")
```

Comment:
Please remove this environment section and the reference to these credentials earlier (line 20)

Reply:
done